src/index.html

```diff
@@ -11,6 +11,58 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    
+    <form 
+      action="#" 
+      data-qa="big" 
+      method="post"
+    >
+      
+      <div class="search big__search">
+
+        <button class="search__button big__search_button">
```

Comment:
<img width="484" alt="Screenshot 2023-09-15 at 10 05 09" src="https://github.com/mate-academy/layout_search-bar-airbnb/assets/53228259/7101befa-0f77-469e-957c-a84b3078bd37">
you failed most tests because of this, try to delete button and add margins to image related to design